crates/sovereign-sdk/adapters/risc0-bonsai/src/host.rs

```diff
@@ -460,17 +459,19 @@ impl<'a> ZkvmHost for Risc0BonsaiHost<'a> {
 
                         // now we convert the snark_receipt to a full receipt
 
-                        let full_snark_receipt = Receipt {
-                            inner: InnerReceipt::Compact(CompactReceipt {
-                                seal: snark_receipt.snark.to_vec(),
-                                claim: receipt
-                                    .get_claim()
-                                    .expect("stark_2_snark error, receipt claim"),
-                            }),
-                            journal: risc0_zkvm::Journal {
-                                bytes: snark_receipt.journal,
-                            },
-                        };
+                        // FIXME:
```

Comment:
1. I would explain why this needs fixing to start with.
2. Maybe create an issue about this and link it here for future reference. That is, if we don't plan on fixing this within this PR.